packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -10,7 +10,7 @@ import { LinkType } from '../../../lib/types';
 import { CreateCompleteResetPasswordLink } from '../../../models/reset-password/verification/factory';
 import { useFinishOAuthFlowHandler } from '../../../lib/oauth/hooks';
 import AppLayout from '../../../components/AppLayout';
-import { CardHeader } from '@material-ui/core';
+import CardHeader from '../../../components/CardHeader';
```

Comment:
The CardHeader reference won't  be needed after #15820 lands. Since we always show the success screen instead of redirecting, we no longer have to worry about handling oauth errors.